code/switch.js

```diff
@@ -0,0 +1,215 @@
+// Start here
+
+// Step 1 - Welcome and introduction
+// Your code goes here
+alert(
+  `Welcome to the Cheesiest Pizzeria. 
+Ready to order some delicious food?
+  
+  Click 'OK' to begin.`
+)
+
+
+const nameInput = prompt(
+  `What is your name?`
+)
+
+switch (nameInput) {
+  case "":
+    alert(`No point in cheating here, we need your actual name`)
+    exit(1)
+  break
+  default: 
+  alert(`Welcome ${nameInput}!
+      
+  Let's begin.
+  We'll make sure to fill your belly!`)
+
+}
+console.log(nameInput)
+
+// Step 2 + 3, Food choice + subtype
+// Your code goes here
+
+const foodChoice = parseInt( prompt(
+  `What would you like to order? 
+  
+  Please choose a number from our meny:
+  1 - Delicious Pizza
+  2 - Pasta
+  3 - Salad`
+)
+)
```

Comment:
use a formatter to have the correct spacing and indentation. Indentation helps readability and has a semantic usage, it indicates a block of code 👍